### DISCUSSION TITLE
Welcome to COS Discussions!

### DISCUSSION THREAD

**Discussion by ldeniau** [CObjectSystem/COS#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jll63** [comment]
Hi Laurent. I am one of your neighbors on [the Multiple Dispatch wikipedia page](https://en.wikipedia.org/wiki/Multiple_dispatch) - [YOMM2](https://github.com/jll63/yomm2), that's me. I have never actually used COS, but I have been following your project for years, as I have a long time fascination for extending computer languages. My very first language was
[FORTH](https://en.wikipedia.org/wiki/Forth_(programming_language)). I am glad to see that COS is alive and well, and I mention it any time I get a chance.
